.github/workflows/nightly_macos_x86_64.yml

```diff
@@ -16,7 +17,7 @@ jobs:
         run: ./ci/write_version.sh
           
       - name: execute rust tests
-        run: cargo test --release --locked -- --skip opaque_wrap_function --skip bool_list_literal --skip platform_switching_swift --skip swift_ui --skip str_append_scalar
+        run: cargo test --release --locked -- --skip opaque_wrap_function --skip gen_list::bool_list_literal --skip platform_switching_swift --skip swift_ui --skip gen_str::str_append_scalar --skip gen_tags::phantom_polymorphic_record
```

Comment:
We already skipped this test at [macos_x86_64.yml](https://github.com/roc-lang/roc/blob/main/.github/workflows/macos_x86_64.yml) but I forgot to update it here as well.